dhis-2/dhis-support/dhis-support-external/src/main/java/org/hisp/dhis/external/conf/ConfigurationKey.java

```diff
@@ -83,6 +83,7 @@
     REDIS_PASSWORD( "redis.password", "", true ),
     REDIS_ENABLED( "redis.enabled", "false", false ),
     REDIS_USE_SSL( "redis.use.ssl", "false", false ),
+    LEADER_TIME_TO_LIVE( "leader.time.to.live.minutes", "2", false ),
```

Comment:
Is this related to the cron of the leader election job? If so, the "format"/cron of the job can be static

Reply:
The format of the leader cron job is static in SchedulerStart.java
```
private final String LEADER_JOB_CRON_FORMAT = "0 0/%s * * * *";
```
We would want the minutes to be externally configured via the dhis.conf. The value given to "leader.time.to.live.minutes" will be replaced in the %s in the cron expression. Default would be every 2 minutes.

Reply:
Ok